project.clj

```diff
@@ -35,7 +35,7 @@
                  [org.cyverse/async-tasks-client "0.0.3"]
                  [org.cyverse/clj-icat-direct "2.9.0"]
                  [org.cyverse/clj-jargon "2.8.9"]
-                 [org.cyverse/clojure-commons "3.0.5"]
+                 [org.cyverse/clojure-commons "3.0.6-SNAPSHOT"]
```

Comment:
I'll change this when the new release of `org.cyverse/clojure-commons` is published.